pkg/cloudevents/transport/stan/options.go

```diff
@@ -0,0 +1,34 @@
+package stan
+
+import "github.com/nats-io/stan.go"
+
+type Option func(*Transport) error
+
+func applyDefaultOptions(t *Transport) error {
+	t.Subscriber = &StandardSubscriber{}
+	return nil
+}
+
+// WithEncoding sets the encoding for STAN transport.
+func WithEncoding(encoding Encoding) Option {
```

Comment:
You don't need this one too: if you want to have specific version encoding/translation, use `binding/transformers/WithVersion`

Reply:
If you need it for implementing the `Transport`, look at how it works in `amqp`